tcms/bugs/tests/test_admin.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+from django.urls import reverse
+from tcms.tests import LoggedInTestCase
+from tcms.utils.permissions import initiate_user_with_default_setups
+
+
+class TestBugAdmin(LoggedInTestCase):
+    '''
+    Class to test Admin view bugs-related functionalities
+    '''
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        initiate_user_with_default_setups(cls.tester)
+
+    def test_user_being_redirected_to_add_bug_form(self):
```

Comment:
Naming ! `test_add_view_redirects_to_new_bug_page`

Reply:
Changed name of this one in new commit